### PR TITLE
Add CI configuration files

### PR DESCRIPTION
**Pull request by mvsusp**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: buildspec-unit.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+    - pip install --upgrade pip --quiet
```
Comment (jesterhazy) [inline buildspec-unit.yml]: quiet hides useful debug info

File: buildspec-unit.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    - pip install tox --quiet
+    - tox -e ${TOX_ENVLIST} -- test/unit
```
Comment (jesterhazy) [inline buildspec-unit.yml]: this project doesn't have a tox config file

Comment (mvsusp) [reply buildspec-unit.yml]: The tox file is located here https://github.com/aws/sagemaker-tensorflow-container/blob/script-mode/tox.ini

File: buildspec-unit.yml (hunk 1 above)
Comment (mvsusp) [reply buildspec-unit.yml]: I carefully try to keep only useful debug information. I did not consider upgrading pip useful. I am happy to remove if you have any specific considerations.

**Pull request merged**